packages/shared/tailwind/overlay.js

```diff
@@ -1,40 +0,0 @@
-const colors = require('./colors');
```

Comment:
Is this now moved to `packages/shared/tailwind/colors/overlay.ts`?

Reply:
So the old one became `packages/shared/tailwind/overlay.ts`
And `packages/shared/tailwind/colors/overlay.ts` is a new one (with the new naming already)